modules/nixos/gnome/default.nix

```
@@ -24,7 +24,7 @@ in
     systemd.services."getty@tty1".enable = false;
     systemd.services."autovt@tty1".enable = false;
 
-    programs.gnupg.agent.pinentryFlavor = mkDefault "gnome3";
+    programs.gnupg.agent.pinentryPackage = mkDefault "pkgs.pinentry-gnome3";
```

Quotes should be removed to denote a package instead of a string
```suggestion
    programs.gnupg.agent.pinentryPackage = mkDefault pkgs.pinentry-gnome3;
```